### PR TITLE
BAH-2367 | Dockerisation and Helm charts

### PR DESCRIPTION
**Pull request by mohan-13**
- Added Dockerfile and Helm chart 
- Refactored Github Actions workflow to build and publish docker image
- Added download_artifact to download and bundle hip-omod
- Refactored `registartion/extenstion.json` with the updated path for abha-id verfication pop-up. Refer [this PR](https://github.com/BahmniIndiaDistro/ABHA-Verification/pull/45).
- Updated paths to custom sql files in reports.json

### CODE REVIEW

File: .github/workflows/build_upload.yml [hunk 1]
```diff
@@ -2,58 +2,84 @@ name: Build and Upload
...
-      - name: Upload Artifact
-        uses: actions/upload-artifact@v2
+      - name: Set env.ARTIFACT_VERSION
+        run: |
+          wget -q https://raw.githubusercontent.com/Bahmni/bahmni-infra-utils/main/setArtifactVersion.sh && chmod +x setArtifactVersion.sh
+          ./setArtifactVersion.sh
+          rm setArtifactVersion.sh
+      - name: Download hip module omod
+        run: |
+          sh .github/download_artifact.sh openmrs-module-hip Package ${{secrets.BAHMNI_PAT}} .
```
Comment (gsluthra) [inline .github/workflows/build_upload.yml]: Will this command "sh <arguments>" print the secret when the action is executed?

Comment (mohan-13) [reply .github/workflows/build_upload.yml]: No Gurpreet. It will not. You will only see a `***` in place of any Github Secret

**Review by N0-man — APPROVED** [review]
We should move `download_artifact.sh` into `bahmni-infra-utils`. I am fine if you want to track the cleanup for all repo with a separate story

**Comment by mohan-13** [comment]
> We should move `download_artifact.sh` into `bahmni-infra-utils`. I am fine if you want to track the cleanup for all repo with a separate story

Yup. We will do that as part of seperate story and the [this](https://bahmni.atlassian.net/browse/BAH-2435) is the JIRA.

**Pull request merged**